atmosphere/tasks/constants.py

```diff
@@ -440,3 +440,5 @@
 HELM_RELEASE_DESIGNATE_NAME = "designate"
 
 HELM_RELEASE_HEAT_NAME = "heat"
+
+HELM_RELEASE_MAGNUM_NAME = "magnum"
```

Comment:
is this being used anywhere?

Reply:
https://github.com/vexxhost/atmosphere/pull/217/files#diff-8ae6ef538f496abc69c5cbbd4923c9dce3c4f7a54b985f851609fe3248b8b481R138-R140

Reply:
no need for that, since we are creating it elsewhere, see comment above.